kazoo/recipe/lock.py

```diff
@@ -197,13 +190,12 @@ def acquire(self, blocking=True, timeout=None, ephemeral=True):
                 )
             except RetryFailedError:
                 pass
-            except KazooException:
+            except KazooException as exc:
                 # if we did ultimately fail, attempt to clean up
-                exc_info = sys.exc_info()
                 if not already_acquired:
                     self._best_effort_cleanup()
                     self.cancelled = False
-                six.reraise(exc_info[0], exc_info[1], exc_info[2])
+                raise exc
```

Comment:
Can't we just `raise` here since it seems we are not making use of `exc`?

Reply:
That's very fair. Forgot you can do that.